yarn/datadog_checks/yarn/yarn.py

```diff
@@ -203,25 +216,34 @@ def _yarn_app_metrics(self, rm_address, app_tags, addl_tags):
         """
         Get metrics for running applications
         """
-        metrics_json = self._rest_request_to_json(rm_address, YARN_APPS_PATH, addl_tags, states=YARN_APPLICATION_STATES)
+        metrics_json = self._rest_request_to_json(rm_address, YARN_APPS_PATH, addl_tags)
 
         if metrics_json and metrics_json['apps'] is not None and metrics_json['apps']['app'] is not None:
-
             for app_json in metrics_json['apps']['app']:
+                tags = self._get_app_tags(app_json, app_tags, addl_tags)
```

Comment:
Seems you don't need to pass `addl_tags`.

Could be: 
```suggestion
                tags = self._get_app_tags(app_json, app_tags) + addl_tags
```


or 

```suggestion
                tags = self._get_app_tags(app_json, app_tags)
                tags.append(addl_tags)
```